consensus/types/src/blob_sidecar.rs

```diff
@@ -93,6 +94,16 @@ impl<T: EthSpec> BlobSidecar<T> {
         Self::default()
     }
 
+    pub fn random<R: Rng>(rng: &mut R) -> Self {
+        let mut blob_bytes = Vec::with_capacity(T::Kzg::BYTES_PER_BLOB);
+        rng.fill_bytes(&mut blob_bytes);
```

Comment:
good shout!

Reply:
I realized the test where I'm using this doesn't interpret blob bytes at all, so I'm opting to just use `BlobSidecar::default` there since the blob size is fixed. I did a litter refactor here to make the code you linked a bit more accessible to anyone trying to test with random valid blobs.